src/software/ai/hl/stp/play/halt_play_test.cpp

```diff
@@ -34,6 +34,7 @@ TEST(StopPlayTest, test_stop_play_returns_correct_tactics)
         {
             StopTactic *unused;
             unused = dynamic_cast<StopTactic *>(t.get());
+            UNUSED(unused);
```

Comment:
So I'm not sure how this even compiles, but it seems like we're using the `UNUSED` macro from `test_util` here?

Reply:
yes from test_util, do you have a better suggestion?

Reply:
Oh sorry, thought this was `halt_play` not `halt_play_test` (hence how it's able to compile), my bad!